frontend/src/colors.js

```diff
@@ -0,0 +1,17 @@
+import {
+  READY,
+  BLOCKED,
+  COMPLETE,
+  IN_REVIEW,
+  IN_PROGRESS,
+  REVIEW_FEEDBACK,
+} from "./constants";
+import { colors } from "@material-ui/core";
+export const cardColors = {
+  [COMPLETE]: colors.yellow[600],
```

Comment:
where else are these colours used? Surely they should be imported in more than one place? Where did you get the colours from originally? Are the colours still defined twice?

Reply:
@sheenarbw I was not working with `src/components/widgets/CardStatusChip.js` 

Does the issue require me to fix all the colors and not only draw the correct colors in the pie chart?